packages/client/src/index.ts

```diff
@@ -360,5 +350,39 @@ export default class LogtoClient {
     const accessTokenKey = buildAccessTokenKey();
     const expiresAt = Date.now() / 1000 + expiresIn;
     this.accessTokenMap.set(accessTokenKey, { token: accessToken, scope, expiresAt });
+    this.saveAccessTokenMap();
+  }
+
+  private saveAccessTokenMap() {
+    if (!this.logtoConfig.persistAccessToken) {
+      return;
+    }
+
+    const data: Record<string, AccessToken> = {};
+
+    for (const [key, accessToken] of this.accessTokenMap.entries()) {
+      // eslint-disable-next-line @silverhand/fp/no-mutation
+      data[key] = accessToken;
+    }
+
+    this.adapter.storage.setItem('accessToken', JSON.stringify(data));
+  }
+
+  private loadAccessTokenMap() {
+    const raw = this.adapter.storage.getItem('accessToken');
+
+    if (!raw) {
+      return;
+    }
+
+    try {
+      const json: unknown = JSON.parse(raw);
+      assert(json, LogtoAccessTokenMapSchema);
+      this.accessTokenMap.clear();
```

Comment:
Should we merge instead of override?

Reply:
I think override is OK since the storage is for current `clientId` only.